src/get-started/install.md

```diff
@@ -66,6 +66,24 @@ with its compilation in certain configurations. If you have encountered a proble
 with installing Exonum on Windows, you may ask around on [Gitter](https://gitter.im/exonum/exonum)
 or file an issue on [GitHub](https://github.com/exonum/exonum/issues).
 
+## Adding environment variables
+
+This variables needed for linking with the `rocksdb` and `snappy` libraries.
+
+### macOS
```

Comment:
Nitpick: Inconsistent capitalization ("macOS" vs "MacOS" in the "Dependencies" section).